UN-OCHA/unocha-site
# UNO-601 Performance improvements

I ran some tests using Google Lighthouse to compare D7 to D10 and it surfaced some actionable items.
1. I'd set the Hero image to lazy, it should remain as eager. This will prevent it from blocking the page render.
3. Add width and height properties to logo img html element to prevent it being a source of layout shift.
4. Image media items using Card view mode are not using webp nor responsive images styles. I've added a responsive style for Card with associated image styles using 3:2 aspect ratio, and set to use webp. The webp will prevent the "Serve images in next-gen formats" issue. However, the First item in the Latest news and stories paragraph is problematic. Since it is usually at least twice the dimensions of the other card thumbnails, the images need to be as large as its largest width, despite the majority of card thumbnails being half the size. This will result in the Performance audits picking up on the absence of "Properly sized images" for pages where the Latest news and stories paragraph exists (Homepage, Region, Response) and where the News and Stories paragraph type is used (Latest news, Stories list pages).

### To test
1. In Chrome, run a Performance lighthouse test on https://www.unocha.org
2. Check out this branch and import the config
3. I think Stage file proxy needs to be disabled for the new images to generate
4.  `drush image:flush` to generate new images
5. In Chrome, run a Performance lighthouse test and notice the items mentioned above are no longer included in the report.
